### ISSUE TITLE
[Feature Request] Custom shell

### ISSUE THREAD

**Issue by mattthhh** [Melv1no/Flow.Launcher.Plugin.easyssh#2]
It would be nice to get the opportunity to change the default terminal emulator that start the ssh process (such as Kitty, or CMD prompt, or Powershell by default)

**Comment by Melv1no** [comment]
Hello @mattthhh, how are you ?

Something like that ?
![image](https://github.com/user-attachments/assets/8a873593-2e6a-4448-a84f-761a5cd7fb3a)

**Comment by mattthhh** [comment]
Yes quite nice (maybe it was already implemented and i missed it), maybe we can also specify the path of the exe that will be run 😀

**Comment by Melv1no** [comment]
I'm currently developing this at your request.

I'm just adding the ability to manually specify the path of the executable.

**Comment by Vaso73** [comment]
How does it look with custom shell??? I could use that too... :-)

**Comment by Vaso73** [comment]
> Yes quite nice (maybe it was already implemented and i missed it), maybe we can also specify the path of the exe that will be run 😀

hi Matth... :-), I want to ask if the ssh shell setup is already working and if you are already using it, if it is fully functional?
